src/super_gradients/training/datasets/dataset_interfaces/dataset_interface.py

```diff
@@ -746,10 +764,69 @@ def __init__(self, dataset_params=None, cache_labels=False, cache_images=False):
                                                 sample_loading_method=val_sample_method,
                                                 cache_labels=cache_labels,
                                                 cache_images=cache_images,
-                                                augment=False)
+                                                augment=False,
+                                                class_inclusion_list=class_inclusion_list)
 
         self.classes = train_sets[1].classes
         self.trainset = ConcatDataset(train_sets)
         self.trainset.collate_fn = train_collate_fn
         self.valset = testset2007
+
+        # ADDING USED ATTRIBUTES FROM PASCAL TO THE CONCATENATED DATASET
         self.trainset.classes = self.classes
+        self.trainset.img_size = self.dataset_params.train_image_size
+        self.trainset.img_files = img_files
+        self.trainset.cache_labels = cache_labels
+        self.trainset.exif_size = train_sets[1].exif_size
+        self.trainset.labels = labels
+
+    def download_pascal(self, delete=True):
+        """
+        Downloads Pascal dataset in YOLO format.
+
+        :param: delete: whether to delete the downloaded zip file after extracting the data (default=True).
+
+        Source: https://github.com/ultralytics/yolov5/blob/master/data/VOC.yaml
+        """
+
+        def convert_label(path, lb_path, year, image_id):
+            def convert_box(size, box):
+                dw, dh = 1. / size[0], 1. / size[1]
+                x, y, w, h = (box[0] + box[1]) / 2.0 - 1, (box[2] + box[3]) / 2.0 - 1, box[1] - box[0], box[3] - box[2]
+                return x * dw, y * dh, w * dw, h * dh
+
+            in_file = open(path / f'VOC{year}/Annotations/{image_id}.xml')
```

Comment:
files are not properly closed. please use:
`with open(lb_path, 'w') as out_file`

Reply:
and add 'path' into the string formating

Reply:
done